cdc/entry/schema_builder.go

```diff
@@ -169,13 +170,14 @@ func (b *StorageBuilder) Build(ts uint64) (*Storage, error) {
 	c := b.baseStorage.Clone()
 	b.baseStorageMu.Unlock()
 
-	err := retry.Run(func() error {
-		err := c.HandlePreviousDDLJobIfNeed(ts)
-		if errors.Cause(err) != model.ErrUnresolved {
-			return backoff.Permanent(err)
-		}
-		return err
-	}, 5)
+	err := retry.Run(10*time.Millisecond, 5,
```

Comment:
this retry will be put in `HandlePreviousDDLJobIfNeed` in https://github.com/pingcap/ticdc/pull/374. Just want to know why this retry limit is changed to 10ms